src/content/app/tools/blast/views/blast-submission-results/components/blast-results-per-sequence/BlastResultsPerSequence.tsx

```diff
@@ -39,7 +41,7 @@ type BlastResultsPerSequenceProps = {
     value: string;
   };
   species: Species[];
-  blastResults: BlastResult[];
+  blastResults: Array<BlastJob & { data: NonNullable<BlastJob['data']> }>;
```

Comment:
`BlastJobResultsFromAPI`? 

Also, we are not using omitting `data` from `BlastJob` here. I think it should be consistent with the definition we used [here](https://github.com/Ensembl/ensembl-client/pull/827/files#diff-15a9df27cf27aa77e289300aa0eb1e20eb87b90a5be68e7c49b401f6a09ed008R67).